modular_skyrat/modules/tribal_extended/code/weapons/sword.dm

```diff
@@ -17,3 +17,7 @@
 	attack_verb_continuous = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	block_chance = 0
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "fire" = 100, "acid" = 50)
+
+/obj/item/claymore/bone/melee_attack_chain(mob/user, atom/target, params)
+	. = ..()
+	user.changeNext_move(CLICK_CD_MELEE * 0.8)
```

Comment:
Need an empty line at the end of this file.

Also... I suppose this works, but it makes me feel like it could have other side-effects, but, eh, I suppose it'll be fine.

Reply:
Done.